src/main/java/rife/continuations/instrument/TypesMethodVisitor.java

```diff
@@ -820,21 +847,7 @@ public void visitMaxs(int maxStack, int maxLocals) {
 
     private void processInstructions(TypesNode node) {
         // set up the context for the node
-        TypesContext context = null;
```

Comment:
Why was this extracted and moved away from the logical context?

Reply:
Same as below.

Reply:
It's much harder to understand if it's somewhere else at a different location.